olive/passes/onnx/insert_beam_search.py

```diff
@@ -123,6 +153,16 @@ def chain_model(
         )
         graph_outputs = [sequences]
 
+        if options["fp16"] and version_1_16:
```

Comment:
should this part be optional? Or is it always recommended?

Reply:
Good catch. It should be used always since ort 1.16.0